proto_logencoder.go

```diff
@@ -76,6 +76,7 @@ func (lfe *grpcLogFieldEncoder) EmitObject(key string, value interface{}) {
 	lfe.emitSafeKey(key)
 	jsonBytes, err := json.Marshal(value)
 	if err != nil {
+		onEvent(newEventUnsupportedValue(key, value))
```

Comment:
The user is going to want the error, I think.

Reply:
Good point.